tests/src/test/scala/cats/tests/RepresentableSuite.scala

```diff
@@ -0,0 +1,43 @@
+package cats.tests
+
+import cats.Representable._
+import cats.data.Kleisli
+import cats.laws.discipline.RepresentableTests
+import cats.laws.discipline.eq._
+import cats.laws.discipline.arbitrary._
+import cats.{Eq, Id, Representable}
+import org.scalacheck.Arbitrary
+
+class RepresentableSuite extends CatsSuite {
+
+  checkAll("Id[Int] <-> Unit => String", RepresentableTests[Id, Unit].representable[String])
+
+  checkAll("String => Int <-> String => Int", RepresentableTests[String => ?, String].representable[Int])
+
+  type Pair[A] = (A, A)
+  checkAll("Pair[String, String] <-> Boolean => String", RepresentableTests[Pair, Boolean].representable[String])
+
```

Comment:
can we check the monad/comonad laws on a Representable here?

Reply:
I added a check for Bimonad